src/main/java/monday/storage/Storage.java

```diff
@@ -0,0 +1,74 @@
+package monday.storage;
+
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+
+import monday.task.Task;
+
+
+/**
+ * Class for handling the storage of tasks in a file.
+ */
+public class Storage {
```

Comment:
It seems intuitive to me to rename this to something more specific, eg `TaskArrayListStorage`, to reflect its intention to be a lot more specific than a general object storage (which could work, but would be implemented through a generic, perhaps).

Reply:
Thanks for the suggestion, couldn't really think of a good name back then, thanks for providing a alternative naming.